scripts/ct.yaml

```diff
@@ -5,4 +5,5 @@ chart-dirs:
   - incubator
 chart-repos:
   - "jetstack=https://charts.jetstack.io"
+  - "fairwinds-incubator=https://charts.fairwinds.com/incubator"
```

Comment:
You're going to want to add stable here too. Goldilocks is moving to stable

Reply:
Done